app/src/main/java/com/example/returnpals/composetools/PickupDateUI.kt

```diff
@@ -91,8 +78,11 @@ fun PickupDateUI(
             )
 
             DateSelector(
```

Comment:
Looking back, do you think we should separate the logic and UI elements more in DateSelector function?
As another way to solve this problem, we make the UI layer expose the buttons to a state layer that holds the Date information and updates the UI layer.  We structure DateSelect to not worry about updating the Date and pass that to a higher function. DateSelect would then only need to read the information from Date.
WheelSelector(
Change this>   onClickPrevious = { onChangeDate(date.minusMonths(1)) },
to something like >  onClickPrevious = { onChangeDateMinus() }

and then pass that parameter onChangeDateMinus()  up
fun DateSelector( 
        onChangeDateMinus: () -> Unit,
this way the composable function is receiving the date information and not updating it. I think this may be better than my other comment idea to solve this problem